assemblies/karaf/pom.xml

```diff
@@ -63,7 +50,7 @@
   <build>
     <resources>
       <resource>
-        <directory>${project.basedir}/src/main/filtered-resources</directory>
```

Comment:
Why the change in the name of the directory? Are the resources no longer filtered?